csharp/ql/src/utils/model-generator/internal/CaptureModelsSpecific.qll

```diff
@@ -16,14 +17,25 @@ module TaintTracking = CS::TaintTracking;
 
 class Type = CS::Type;
 
+/**
+ * Holds if `api` is an override of `GetHashCode` or `Equals`.
+ */
+private predicate isIrrelevantObjectOveride(CS::Callable api) {
```

Comment:
Ahh, sorry, I didn't notice earlier, but "Overide" also has a typo.

Reply:
OMG - TGIF! Thank you 👍